scripts/performance_benchmark.py

```
@@ -0,0 +1,210 @@
+        results = benchmark_table_converter()
+        
+        # Test limit effectiveness
+        benchmark_with_limit()
+        
+        print("\n✅ Benchmark completed successfully!")
+        
+    except KeyboardInterrupt:
+        print("\n⚠️  Benchmark interrupted by user")
+    except Exception as e:
+        print(f"\n❌ Benchmark failed: {e}")
+        traceback.print_exc()
```

Ruff (0.11.9)

Undefined name `traceback`

(F821)